awx/main/utils/common.py

```diff
@@ -750,7 +750,7 @@ def convert_cpu_str_to_decimal_cpu(cpu_str):
     return max(0.1, round(cpu, 1))
 
 
-def get_corrected_cpu(cpu_count):  # formerlly get_cpu_capacity
+def get_corrected_cpu(cpu_count, obj):  # formerlly get_cpu_capacity
```

Comment:
I would prefer a change to the method signature that is backward-compatible like adding a kwarg like `, is_execution=False` which can be flipped to True when necessary.

Also, I don't see any corresponding change to set the new option, presumably in the task code for running health checks.